isis/tests/FunctionalTestsGetsn.cpp

```diff
@@ -0,0 +1,151 @@
+#include <iostream>
+#include <sstream>
+
+#include <QIODevice>
+#include <QTextStream>
+#include <QStringList>
+#include <QFile>
+
+#include "getsn.h"
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "Cube.h"
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/getsn.xml").expanded();
+static QString FROM_PARAM = "FROM=data/defaultImage/defaultCube.pvl";
```

Comment:
This implicitly links this test to the fixture. If we change the location of the fixture data or the name of the label file it will need to be updated here as well. The path to the label should either be a member of the fixture or a static variable that is defined in the Fixture code so that it only needs to be updated in a single place.

Reply:
Considering that he is using the interface with the cube being passed in, from doesn't have to even be part of the args and it should be able to be eliminated altogether.

Reply:
I agree, it would be better to use the method that passes in a Cube object. You can use Cube::fileName to get the path for checking the output too.